core/src/cats/cats.h

```diff
@@ -469,7 +469,7 @@ class pathid_cache;
 #define QUERY_HTABLE_PAGES 128
 
 // Current database version number for all drivers
-#define BDB_VERSION 2210
+#define BDB_VERSION 2301
```

Comment:
This is probably open to discussion, but previously the schema was 2000 + 10 * Major + Minor, which would result in 2230 for Bareos 23.0.0.
However, I don't really care as long as it is strictly increasing.

Reply:
Thanks I was trying to understand the pattern behind hit. So I will adapt, but also add the formula as comment, so next contributor will have its life eased ;-)